crates/iroha/tests/fees.rs

```diff
@@ -0,0 +1,89 @@
+use eyre::Result;
+use fees_executor_data_model::parameters::*;
+use iroha::data_model::prelude::*;
+use iroha_executor_data_model::parameter::Parameter as _;
+use iroha_test_network::*;
+mod upgrade;
+use upgrade::upgrade_executor;
+
+#[test]
+fn fees_options_in_updated_executor() -> Result<()> {
+    let (network, _rt) = NetworkBuilder::new().start_blocking()?;
+
+    let test_client = network.client();
+    // Also sets enough fuel based on the profile
+    upgrade_executor(&test_client, "fees_executor")?;
```

Comment:
Just an idea: add `NetworkBuilder::with_executor()` to inject the executor right into the genesis.

```rust
NetworkBuilder::new()
.with_executor("fees_executor")
.start_blocking()?;
```

Reply:
The fuel limits should also be adjusted (in case of debug mode).

Reply:
Yup, that could also be moved into the `NetworkBuidler`.

Reply:
I'm not sure it belongs there. `NetworkBuilder` seems to be logically divided from the `test_samples` which need to know about the sample wasms options.

Reply:
Well, then `with_executor` can accept just `WasmContract`:

```rust
NetworkBuilder::new()
.with_executor(test_samples::load_wasm("fees_executor"))
.start_blocking()?;
``` 

(I mean, that's not really an issue. Just thinking about making `NetworkBuilder` a tool to help with writing concise and expressive tests. `with_executor` seems to be one of a small reusable niceties.)